BUILD

```diff
@@ -127,19 +127,41 @@ using the command
 12. Release
 -----------
 
-You can use maven to do releases by using the maven plugin. First you need to
-ensure all your dependencies are not SNAPSHOTS and that everything is commited
+You can use Maven to do releases by using the Maven release plugin.
 
-Then you can call
+Prerequisites:
+* You must have set your system for automatic login to Github through SSH.
+* You must have a local working copy of gmp. It must have git@github.com:gemini-hlsw/gmp.git as a
+  remote repository. Your master branch must be up to date with the master branch on the gemini-hlsw
+  repository.
+* You also must fork gemini-hlsw/maven-repo.git in Github, and have an updated local working copy.
+
+You can start doing an initial test. In the root of your local working copy of gmp, run the following
+command:
 
 mvn release:clean release:prepare -DdryRun=true
 
 If that works fine you can do the actual release preparation as
 
 mvn release:prepare
 
-Once that is ready the command
+That command will:
+  1. Update all the SNAPSHOT version to final versions.
+  2. Commit the changes to the versions.
+  3. Tag the repository with the new version for GMP (GMP-R<version>)
+  4. Increment all the version numbers and add the SNAPSHOT prefix to them, in preparation for the next
+     development cycle.
+  5. Commit the changes to the versions.
+  6. Push all the changes and the new tag to the gemini-hlsw repository in Github.
+
+Once that is ready you can actually perform the release with the command (you must use the location of your
+local maven-repo working copy):
```

Comment:
Maybe explicit that this refers to the `maven-repo` on github under `geminihlsw`

Reply:
That is already mentioned in the third prerequisite.

Reply:
Ok, I missed that